tests/schemas/api_v3/crossChainMessages.schema.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+
+const regex = require('./regex');
+
+const getCurrentTime = () => Math.floor(Date.now() / 1000);
+
+const block = {
+	id: Joi.string().min(1).max(64).pattern(regex.HASH_SHA256)
+		.required(),
+	height: Joi.number().integer().min(1).required(),
+	timestamp: Joi.number().integer().positive().max(getCurrentTime())
+		.required(),
+	transactionID: Joi.string().min(1).max(64).pattern(regex.HASH_SHA256)
+		.required(),
+};
+
+const CCM_STATUS = [
+	'ok',
+	'module_not_supported',
+	'ccm_not_supported',
+	'channel_unavailable',
+	'recovered',
+];
+
+const crossChainMessageSchema = {
+	moduleCrossChainCommandID: Joi.string().required(),
```

Comment:
Can we please apply the same pattern checks to all the properties that we've applied in `services/gateway/apis/http-version3/methods/crossChainMessages.js`?
```suggestion
	id: Joi.string().pattern(regex.HASH_SHA256).required(),
	moduleCrossChainCommandID: Joi.string().required(),
```